assessment.rb

```diff
@@ -1,12 +1,67 @@
 require 'csv'
 # Assessment answers to:
 
+#ran: rails g model Licensee
+#thenran: rake db:migrate
+
 # question 2:
+Person.create(surname: 'Bishop', given_name: 'Caren', gender: 'f', height: 63,weight: 132, dob: '1943-09-26',eye_color: 'Brown', hair_color: 'Black')
 
 # question 3:
 
+Licensees.transaction do
+          CSV.foreach(Rails.root + 'data/licensees.csv' headers: true) do | Licensee_row |
+
+            Licensee = Licensee_row.to_hash
+            next if Licensee.exists? Licensee
```

Comment:
In these two lines, the variable should be `licensee`, not `Licensee` (which refers to the class).